### PR TITLE
added options to includes all channel scan and 20mhz bw ap

### PR DESCRIPTION
**Pull request by rtrue983**
all channel scan fixes the problem where there a multiple routers with the same SSID that it is trying to connect to and instead of trying to connect to the router it first finds based on channel (could be lowest rssi) it will scan all the channels first and connect to the one with the highest rssi

also added the option to use a 20MHz bandwidth AP

### CODE REVIEW

File: mos.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ config_schema:
   - ["wifi.ap.trigger_on_gpio", "i", -1, {title: "Trigger AP on low GPIO"}]
   - ["wifi.ap.disable_after", "i", 0, {title: "If > 0, will disable itself after the specified number of seconds"}]
   - ["wifi.ap.hostname", "s", "", {title: "If not empty, DNS server will resolve given host name to the IP address of AP"}]
+  - ["wifi.ap.bandwidth_20mhz", "b", false, {title: "enable 20MHz bandwidth AP operation"}]
```
Comment (rojer) [inline mos.yml]: since these are esp32 specific, move them to the appropriate section under conds below

File: mos.yml [hunk 2]
```diff
@@ -44,6 +45,7 @@ config_schema:
   - ["wifi.sta.gw", "s", {title: "Static Default Gateway"}]
   - ["wifi.sta.nameserver", "s", {title: "DNS Server"}]
   - ["wifi.sta.dhcp_hostname", "s", {title: "Host name to include in DHCP requests"}]
+  - ["wifi.sta.all_chan_scan", "b", false, {title: "true to scan all channels or false to perform fast scan"}]
```
Comment (rojer) [inline mos.yml]: please call this `wifi.sta_all_chan_scan` the difference being that it doesn't get triplicated in the sta1 and sta2 sections and it really applies to all scans and is not specific to station profile

Comment (rtrue983) [reply mos.yml]: @rojer the problem I see with that is that wifi cfg isn't available in the `mgos_wifi_dev_setup` function?

Comment (rojer) [reply mos.yml]: `mgos_sys_config_get_wifi_sta_all_chan_scan()`

**Pull request merged**